cmd/docker/main.go

```diff
@@ -126,6 +126,17 @@ func handleFlags() (*CliOptions, bool) {
 		}
 	}
 
+	// load the vch config
+	src, err := extraconfig.GuestInfoSource()
+	if err != nil {
+		log.Errorf("Unable to load configuration from guestinfo")
```

Comment:
I realize this was existing code, but shouldn't we include the original error in that log message and use log.Fatalf to exit(1) rather than continue?